lms/static/sass/bootstrap/_layouts.scss

```diff
@@ -1,8 +1,10 @@
 // LMS layouts
 
 .content-wrapper {
+    margin-top: $baseline;
```

Comment:
A little worried about this margin, most pages handle the spacing issues within the content itself. If you look at the attached image, the left shows what this margin will do to the courseware tabs. Instead, you can add the class "mt-3" to the h3 element in preview_lang_include.html.

For reference on Bootstrap 4 spacing: https://v4-alpha.getbootstrap.com/utilities/spacing/


<img width="1230" alt="screen shot 2017-07-19 at 10 28 38 am" src="https://user-images.githubusercontent.com/3861052/28372697-5ed95310-6c6e-11e7-866f-e312f27203da.png">